Repositories/Readme.md

```diff
@@ -5,9 +5,9 @@
 
 ## Inserting software/data links into your manuscript.
 
-  Current practices include inserting `\footnote` or inline URL links into a manuscript or adding first class data/software references directly into a manuscript's bibliography. AAS Journals recognize all of these options for referencing data or software related to an article, but prefer that the link/archive be made as persistent as possible. Two articles that might aide your decisions about archiving data/sofware include: "Ten Simple Rules for the Care and Feeding of Scientific Data [(10.1371/journal.pcbi.1003542)](dx.doi.org/10.1371/journal.pcbi.1003542)" and "Achieving human and machine accessibility of cited data in scholarly publications [(10.7717/peerj-cs.1)](https://dx.doi.org/10.7717/peerj-cs.1)"
+  Current practices include inserting `\footnote` or inline URL links into a manuscript or adding first class data/software references directly into a manuscript's bibliography. AAS Journals recognize all of these options for referencing data or software related to an article, but prefer that the link/archive be made as persistent as possible. Two articles that might aide your decisions about archiving data/sofware include: "Ten Simple Rules for the Care and Feeding of Scientific Data [(10.1371/journal.pcbi.1003542)](http://dx.doi.org/10.1371/journal.pcbi.1003542)" and "Achieving human and machine accessibility of cited data in scholarly publications [(10.7717/peerj-cs.1)](https://dx.doi.org/10.7717/peerj-cs.1)"
 
-  AAS Journals encourage authors to place data or software products related to an article into a persistent repository linked to the main journal article by a DOI and listed in the main references. See subsequent sections in this tutorial on formatting those references, and inserting them into your manuscript.
+  AAS Journals strongly encourage authors to place data or software products related to an article into a persistent repository linked to the main journal article by a DOI and listed in the main references. See subsequent sections in this tutorial on formatting those references, and inserting them into your manuscript.
```

Comment:
It might become policy if/when we have our own repository (or a contractual relationship with one).